tasks/ceph.py

```diff
@@ -71,10 +72,79 @@ def ceph_log(ctx, config):
             )
         )
 
+    class Rotater:
```

Comment:
Very minor style thing: new python code uses "class Rotater(object):" in order to be a "new style" class.  Doesn't actually matter unless you're doing inheritance etc.